Jenkinsfile

```diff
@@ -148,7 +148,8 @@ stage('prepare') {
     checkout scm
     sh 'git clean -ffdx -e .????????'
     sshagent(['realm-ci-ssh']) {
-      sh 'git submodule update --init --recursive'
+      sh 'git submodule sync && git submodule update --init --recursive'
```

Comment:
I don't think `submodule sync` should ever do anything on CI since we run from a fresh checkout every time?

Reply:
For some reason, this was necessary when I tried changing over to my forked version of catch. Not sure why, maybe due to source caching in CI? However, that is no longer needed, so I'll remove it.